src/core/sdl/window_sdl.cpp

```diff
@@ -89,14 +89,26 @@ bool WindowSDL::initSDL(const CmdLineSetup_t &setup, uint32_t windowInitFlags)
 
     SDL_GL_ResetAttributes();
 
+#if defined(__SWITCH__) /* On Switch, expect the initial size 1280x720 */
+    const int initWindowW = 1920;
+    const int initWindowH = 1080;
```

Comment:
Should it be 1280x720? This requires less scaling. In docked mode, it might upscale to 1920x1080 then downscale to 1280x720. We can also review this later at multires time.

Reply:
I did some tests, and it should keep the 1920x1080, but, when I set 1280x720, when it's upscaled, quality distorts

Reply:
I talked with Yuzu folks about this, and it's better to keep 1920x1080 for the best picture quality, and it's fine when it gets auto-scaled during portable mode.

Reply:
This is a pixel graphics game, so I'd prefer to keep the integer scaling as close to prefect as possible, but let's worry about that during Multires. Right now there is no integer scaling support at all.